content/blog/2016/2016-05-31-Jenkins-Usage-Statistics-Analysis-Intro.adoc

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "GSoC: Jenkins Usage Statistics Analysis Introduction"
+tags:
+-infra_statistics
```

Comment:
Two tags (this and `jenkins_usage`) basically exclusively for this project are a bit much. I can see one tag making sense if there's going to be multiple posts, but two?